bundles/io/org.eclipse.smarthome.io.multimedia.sonos/pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <parent>
+    <groupId>org.eclipse.smarthome.bundles</groupId>
+    <artifactId>io</artifactId>
+    <version>0.7.0-SNAPSHOT</version>
+  </parent>
+
+  <properties>
+    <bundle.symbolicName>org.eclipse.smarthome.io.multimedia.tts.googletranslate</bundle.symbolicName>
+    <bundle.namespace>org.eclipse.smarthome.io.multimedia.tts.googletranslate</bundle.namespace>
+  </properties>
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.eclipse.smarthome.io</groupId>
+  <artifactId>org.eclipse.smarthome.io.multimedia.tts.googletranslate</artifactId>
```

Comment:
Shouldn't the artifact ID be similar to the directory (org.eclipse.smarthome.io.multimedia.tts.googletranslate vs. org.eclipse.smarthome.io.multimedia.sonos)?